### PR TITLE
Added tagprefix doc for AWS and Cloud Costs

### PR DESCRIPTION
**Pull request by jdesouza**
This PR fixes #

## Checklist
* [x] I have signed the CLA
* [x] I have updated/added any relevant documentation

### CODE REVIEW

File: docs/technical-details/reports/aws-costs.md [hunk 1]
```diff
@@ -54,13 +54,25 @@ awscosts:
...
+Tags with special characters must be replaced with underscores. Example: `aws:eks:cluster-name`  should be aws_eks_cluster_name.
+* **tagprefix**: tagprefix is a prefix AWS adds to your tag in order to create Athena column. In this example the column in Athena will be: resource_tags_user_kubernetes_cluster.
+Ex: `KubernetesCluster` (in case of Kops). The column name in Athena has a prefix resource_tags_user_. 
+In the case you are using a tag provided by AWS the prefix can be a little bit different like resource_tags_.
+Ex: if you are using standard tag `aws:eks:cluster-name` from AWS EKS you need to set:
+```yaml
+tagprefix = resource_tags_
+tagkey    = aws_eks_cluster_name
+```
+Athena column in this case is resource_tags_aws_eks_cluster_name
```
Comment (rbren) [inline docs/technical-details/reports/aws-costs.md:75]:
```suggestion
* **tagkey**: tagkey is the AWS tag used on EC2 instances that are nodes in this cluster. Note that AWS applies pascal camel to split the tag name, and replaces special characters with `_`. Example: `aws:eks:cluster-name`  should become aws_eks_cluster_name.
* **tagprefix**: tagprefix is a prefix AWS adds to your tag in order to create an Athena column. Default is `resource_tags_user_`, but may change for different circumstances.
E.g. if you are using the standard tag `aws:eks:cluster-name` from AWS EKS, the Athena column will become `resource_tags_aws_eks_cluster_name`, and you need to set:
```yaml
tagprefix = resource_tags_
tagkey    = aws_eks_cluster_name
```

File: docs/technical-details/reports/aws-costs.md [hunk 2]
```diff
@@ -54,13 +54,20 @@ awscosts:
...
 
 * **database**: the database created on AWS Glue Data
 * **table**: aws cur report name
-* **tagkey**: tag key is the tag used on EC2 to indicate that it's a cluster node. Ex: KubernetesCluster (in case of Kops). The column name in Athena has a prefix resource_tags_user_. Also AWS applies pascal camel to split the tag name. In this example the column in Athena will be: resource_tags_user_kubernetes_cluster.
+* **tagkey**: tagkey is the AWS tag used on EC2 instances that are nodes in this cluster. Note that AWS applies pascal camel to split the tag name, and replaces special characters with `_`. Example: `aws:eks:cluster-name`  should become aws_eks_cluster_name.
+* **tagprefix**: tagprefix is a prefix AWS adds to your tag in order to create an Athena column. Default is `resource_tags_user_`, but may change for different circumstances.
+E.g. if you are using the standard tag `aws:eks:cluster-name` from AWS EKS, the Athena column will become `resource_tags_aws_eks_cluster_name`, and you need to set:
+```yaml
+tagprefix = resource_tags_
+tagkey    = aws_eks_cluster_name
```
Comment (rbren) [inline docs/technical-details/reports/aws-costs.md:70]:
```suggestion
tagkey    = aws_eks_cluster_name
```
```

Comment (rbren) [inline docs/technical-details/reports/aws-costs.md:70]:
```suggestion
tagkey    = aws_eks_cluster_name
````
```

File: docs/technical-details/reports/aws-costs.md [hunk 3]
```diff
@@ -54,13 +54,20 @@ awscosts:
...
 * **tagvalue**: the value associated to the tag for filtering. Ex: production, staging
```
Comment (rbren) [inline docs/technical-details/reports/aws-costs.md:71]:
```suggestion
```yaml
tagprefix = resource_tags_
tagkey    = aws_eks_cluster_name
```
* **tagvalue**: the value associated to the tag for filtering. Ex: production, staging
```

File: docs/technical-details/reports/aws-costs.md (hunk 2 above)
Comment (rbren) [inline docs/technical-details/reports/aws-costs.md:70]:
````suggestion
tagkey    = aws_eks_cluster_name
```
````

File: docs/technical-details/reports/aws-costs.md [hunk 4]
```diff
@@ -54,13 +54,21 @@ awscosts:
...
 
 * **database**: the database created on AWS Glue Data
 * **table**: aws cur report name
-* **tagkey**: tag key is the tag used on EC2 to indicate that it's a cluster node. Ex: KubernetesCluster (in case of Kops). The column name in Athena has a prefix resource_tags_user_. Also AWS applies pascal camel to split the tag name. In this example the column in Athena will be: resource_tags_user_kubernetes_cluster.
+* **tagkey**: tagkey is the AWS tag used on EC2 instances that are nodes in this cluster. Note that AWS applies pascal camel to split the tag name, and replaces special characters with `_`. Example: `aws:eks:cluster-name`  should become aws_eks_cluster_name.
+* **tagprefix**: tagprefix is a prefix AWS adds to your tag in order to create an Athena column. Default is `resource_tags_user_`, but may change for different circumstances.
+E.g. if you are using the standard tag `aws:eks:cluster-name` from AWS EKS, the Athena column will become `resource_tags_aws_eks_cluster_name`, and you need to set:
+```yaml
+tagprefix = resource_tags_
+tagkey    = aws_eks_cluster_name
```
Comment (rbren) [inline docs/technical-details/reports/aws-costs.md:70]:
```suggestion
tagprefix: resource_tags_
tagkey: aws_eks_cluster_name
```

**Review by rbren — APPROVED** [review]
Did a little wordsmithing :)

**Comment by jdesouza** [comment]
> Did a little wordsmithing :)

Thanks! :-D

**Pull request merged**